hub/api/dataset.py

```diff
@@ -59,7 +60,7 @@ def __init__(
                 Use this if you want to specify the storage provider object manually instead of using a tag or url to generate it.
             public (bool, optional): Applied only if storage is Hub cloud storage and a new Dataset is being created. Defines if the dataset will have public access.
             token (str, optional): Activeloop token, used for fetching credentials for Hub datasets. This is optional, tokens are normally autogenerated.
-
+            _tensors: (list, optional): Internal.
```

Comment:
what is this for? can you docstring it

Reply:
also, is there a way we can remove this parameter? this will show up in our API reference and the parameter list is already pretty long

Reply:
+1 on removing it, I haven't really seen unused parameters in any user-facing api. Although we can address this later, removing it won't be trivial.